sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2ScanRelationPushDown.scala

```diff
@@ -282,7 +288,7 @@ object V2ScanRelationPushDown extends Rule[LogicalPlan] with PredicateHelper {
       }
       operation
     case s @ Sort(order, _, operation @ ScanOperation(_, filter, sHolder: ScanBuilderHolder))
-      if filter.isEmpty =>
+        if filter.isEmpty =>
```

Comment:
I feel there should be two indent.

Reply:
it should be 4 spaces. You can check it out in other code files.

Reply:
Ref: https://github.com/apache/spark/blob/fe73039f991ce2c44bc5bb2dc845c735e6959c14/sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala#L493
https://github.com/apache/spark/blob/fe73039f991ce2c44bc5bb2dc845c735e6959c14/sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala#L561
Maybe we could unify the code style.